app/views/users/management.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1>User Management</h1>
+<%= link_to(button_tag("Create User",  class:"btn btn-primary"), new_user_path) %>
+<br><br>
```

Comment:
add a link back to the root page as well?? wouldn't want someone to get trapped here

Reply:
Can do!